modules/extensions/src/data-filter/data-filter.js

```diff
@@ -79,6 +82,97 @@ export default class DataFilterExtension extends LayerExtension {
         }
       });
     }
+
+    const {gl} = this.context;
+    if (attributeManager && extension.opts.countItems) {
+      if (aggregator.isSupported(gl)) {
+        attributeManager.addInstanced({
+          filterIndices: {
+            size: 1,
+            vertexOffset: 1,
+            type: GL.UNSIGNED_BYTE,
+            accessor: (_, {index}) => (index + 1) % 2,
```

Comment:
I didn't get this part. How does this count filtered items? Is this run before or after the actual filtering? A comment here perhaps?

Reply:
Variable-width layers (path, polygon, text etc.) have multiple vertices per object. This attribute is used in the shader so that only one flag is generated for each object.